scripts/anonymize.sql

```diff
@@ -47,11 +48,14 @@ TRUNCATE tidings_watch;
 
 UPDATE auth_user SET
     -- username left alone, because it's public info
-    password = NULL,
+    password = '!',
     email = CONCAT('user-', id, '@example.com'),
     first_name = ROUND(RAND()*1000000),
     last_name = ROUND(RAND()*1000000);
 
+UPDATE account_emailaddress SET
+    email = CONCAT('user-', id, '@example.com');
```

Comment:
I think this shouldn't use the `id` of the account email address row but the user id the email address refers to, to continue have some relation to the data in the `auth_user` table. So this should work:

`UPDATE account_emailaddress SET email = CONCAT('user-', user_id, '@example.com');`